ot/sliced.py

```diff
@@ -0,0 +1,100 @@
+"""
+Sliced Wasserstein Distance.
+
+"""
+
+# Author: Adrien Corenflos <adrien.corenflos@gmail.com>
+#
+# License: MIT License
+
+
+import numpy as np
+
+
+def _random_projections(n_projections, dimension, random_state):
+    """Samples n_projections times dimension normal distributions"""
+    projections = random_state.normal(0., 1., [n_projections, dimension])
+    norm = np.linalg.norm(projections, ord=2, axis=1, keepdims=True)
+    projections = projections / norm
+    return projections
+
+
+def sliced(X_s, X_t, a=None, b=None, n_projections=50, seed=None):
```

Comment:
all parameter `log=False` that will allow returning information (at least the random projection operator)

Reply:
Also you should name the function `sliced_wasserstein_distance`